TerletskayaJE/ConsoleApplication2/ConsoleApplication2/Source.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream> 
+#include <conio.h> 
+using namespace std;
+
+int main()
+
+{
+	setlocale(0, "");
+	float left, right, result;
+	int erorr = 0;
+	cout << "������� ����� ��������: ";
+	cin >> left;
+	cout << "������� ������ ��������: ";
+	cin >> right;
+	char sign;
+	cout << "������� ���� ����� �����������: ";
+	cin >> sign;
+	switch (sign)
+	{
+	case '+': result = left + right;
+		break;
+	case '-': result = left - right;
+		break;
+	case '*': result = left * right;
+		break;
+	case '/':
+		if (right == 0)
+		{
+			cout << "�� ���� ������ ������!";
+			erorr = 1;
+			break;

```

Comment:
Break operator presented here and in "else" branch simultaneously. That means that you can put it right after confidition block, because it will be executed anyway.